rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -2064,19 +2063,23 @@ private Integer createFederation(boolean dryRun) {
         PendingFederation currentPendingFederation = provider.getPendingFederation();
 
         if (currentPendingFederation != null) {
-            return -1;
+            logger.warn("[createFederation] Already existing pending federation.");
```

Comment:
```suggestion
            logger.warn("[createFederation] A pending federation already exists.");
```
Sounds a bit more "human" to me at least